Cargo.toml

```diff
@@ -66,6 +41,7 @@ members = [
   "pallets/rewards",
   "runtime/altair",
   "runtime/centrifuge",
+  "runtime/development",
```

Comment:
I can not tell why, but you can compile development even if it is not on the list. Anyways, adding it here for correctness